core/src/main/kotlin/com/katanox/tabour/configuration/core/Core.kt

```diff
@@ -0,0 +1,10 @@
+package com.katanox.tabour.configuration.core
+
+import com.katanox.tabour.Tabour
+import com.katanox.tabour.consumption.Config
+
+/** Creates a new [Tabour] instance */
+fun tabour(init: Tabour.Configuration.() -> Unit): Tabour =
```

Comment:
Suggestion: I believe it is better to move these methods to Tabour class to keep following SOLID principles

Reply:
not sure what you mean here, can you explain what you mean by solid? These are dsl functions that construct the required classes

Reply:
These two functions do belong to the main class as these are doing initialization operations on tabour class, therefore there is no need to have these two functions in a separate file